user-guides/Privisus/people.md

```diff
@@ -0,0 +1,61 @@
+# Reviewing user guides: People.
+
+In this document, I reviewed the following:
+
+* Check whether someone is on-line
+* Invite another user
+
+-----
+
+## Check whether someonw is on-line
```

Comment:
Replace "someonw" with "someone" :)

Reply:
Hmm, very weird. I definitely have fixed it on the last push.